app/services/proofing/resolution/plugins/residential_address_plugin.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Proofing
+  module Resolution
+    module Plugins
+      class ResidentialAddressPlugin
+        attr_reader :proofer, :sp_cost_token
+
+        def initialize(
+          proofer:,
+          sp_cost_token:
+        )
+          @proofer = proofer
```

Comment:
Responsibility for creating `Proofer` instances is moved into `ProgressiveProofer`, allowing these `*AddressPlugin` classes to be shared between Socure / InstantVerify.